lib/classes/tooltip.js

```diff
@@ -373,7 +373,7 @@ class ToolTip {
     getTipElement() {
         if (!this.$tip) {
             // Try and compile user supplied template, or fallback to default template
-            this.$tip = this.compileTemplate(this.$config.template) || this.compileTemplate(this.constructor.Default.template);
+            this.$tip = ToolTip.compileTemplate(this.$config.template) || ToolTip.compileTemplate(this.constructor.Default.template);
```

Comment:
We cant do this, otherwie Popover becomes a tooltip..  This needs to be `this.` references